bin/control.bat

```diff
@@ -224,3 +195,53 @@ del %RESTART_SUCCESS_FILE%
 if not "%NO_PAUSE%" == "1" pause
 
 goto :eof
+
+:getJavaSpecificOpts
```

Comment:
Do we need to define the getJavaSpecificOpts function here? It should be already defined in the jvmdefaults.bat.